src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -1604,6 +1602,37 @@ public void deleteNode(UUID studyUuid, UUID nodeId, boolean deleteChildren, Stri
         notificationService.emitElementUpdated(studyUuid, userId);
     }
 
+    @Transactional
+    public void stashNode(UUID studyUuid, UUID nodeId, boolean stashChildren, String userId) {
+        AtomicReference<Long> startTime = new AtomicReference<>(null);
+        startTime.set(System.nanoTime());
+        DeleteNodeInfos deleteNodeInfos = new DeleteNodeInfos();
```

Comment:
I don't see how DeleteNodeInfos is needed for this use case, if i'm not mistaken it's used during a node's deletion to keep track of its modifications, network, and various results after it's been deleted from the db but in case of a stash is it really useful ?